src/main/kotlin/com/exactpro/th2/actuibackend/protobuf/ProtoSchema.kt

```diff
@@ -31,7 +35,33 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 import mu.KotlinLogging
 
-data class JsonSchema(val schemaName: String, val schema: String)
+data class JsonSchema private constructor(val schemaName: String, val schema: JsonNode) {
+    companion object {
+        private val parentEventIdField = "parentEventId"
+        private val refValues = "\$ref"
+        private val definitions = "#/definitions/"
+
+        private fun deleteParentEventId(schema: JsonNode) {
```

Comment:
maybe transform to extension method for JsonNode

Reply:
it seems to me that no. it will only be used in one place and may be removed in the future